src/middlewared/middlewared/plugins/enclosure_/map2.py

```diff
@@ -1,71 +1,59 @@
-from .slot_mappings import get_mapping_info
-
-
-def map_enclosures(enclosures):
-    mapped = [{'id': 'mapped_enclosure_0', 'elements': {'Array Device Slot': {}}}]
-    jbods = []
-    for enclosure in enclosures:
-        if not enclosure['controller']:
-            # we're on a platform that has a JBOD attached so we
-            # want to place these in another list so we don't lose
-            # them during the mapping process
-            jbods.append(enclosure)
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def to_ignore(enclosure, model):
```

Comment:
We should probably return False on default case here so that type returned remains consistent.

Reply:
Good catch, fixed it.